.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: flake8 Lint
+
+on: [pull_request]
+
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v2
+      - name: Set up Python environment
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: flake8 Lint
+        uses: py-actions/flake8@v1
```

Comment:
Does this automatically run flake8 linting without the "run:" keyword? I am wondering where exactly flake8 is taking action and trying to locate the resulting output message from lint.

Reply:
Hmm yea it doesn't need the run keyword. It's all wrapped within the `uses: py-actions/flake8@v1` line I believe. The resulting output message from lint can be seen from the checks tab.